src/main/resources/default/assets/wondergem/autocomplete/multiselect.js

```diff
@@ -208,25 +224,42 @@ var multiSelect = function (args) {
             $('#' + args.id).find('.tokenfield').removeAttr("tabindex");
         });
 
+        // Special handling for singleSelects with keepInputOnSelect = true
+        // The tokenfield doesn't trigger any changes if the input was cleared manually, so we need to do this by hand
+        if (args.keepInputOnSelect) {
+            tokenfield.getTokenfieldInputField().on('keyup', function (e) {
+                if (tokenfield.getInput().trim().length === 0 && e.keyCode === sirius.keys.KEY_ENTER) {
+                    tokenfield.getTokenfieldInputField().blur();
```

Comment:
Wieso den "blur"?

Reply:
Weil das Tokenfeld bei leerer Eingabe und Enter nichts macht, in den Fall sollte er aber rauspringen